modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/rest-openapi.yaml

```diff
@@ -6342,6 +6392,107 @@ paths:
                       - HeadlessCommerceAdminCatalog.read
             summary: Gets a related product by ID.
             tags: ["ProductChannel"]
+    "/product-configuration-list-account-groups/{productConfigurationListAccountGroupId}":
+        delete:
+            parameters:
+                - in: path
+                  explode: false
+                  name: productConfigurationListAccountGroupId
+                  required: true
+                  schema:
+                      description:
+                          The resource Id
+                      example: 30130
+                      format: int64
+                      minimum: 0
+                      type: integer
+                  style: simple
+            responses:
+                204:
+                    content:
+                        application/json: {}
+                        application/xml: {}
+                401:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                    description:
+                        Authentication information is missing or invalid
+            security:
+                - BasicAuth: []
+                - BearerAuth: []
+                - OAuth2:
+                      - CommerceOpenApiAdmin.write
+            summary: Deletes a product configuration list account group by ID.
+            tags: ["ProductConfigurationListAccountGroup"]
+    "/product-configuration-list-account-groups/{productConfigurationListAccountGroupId}/account-group":
```

Comment:
also here I think we can send these 2 fields along with the other entity and remove this endpoint